src/Compute/Compute/help/Set-AzVMCustomScriptExtension.md

```diff
@@ -295,16 +459,31 @@ This cmdlet adds the custom script extension for the virtual machine that this p
 
 ```yaml
 Type: System.String
-Parameter Sets: (All)
+Parameter Sets: ByNameWithContainerAndFileNamesParameterSet, ByNameWithFileUriParameterSet
 Aliases: ResourceName
 
 Required: True
-Position: 1
+Position: Named
```

Comment:
Hmm looked like this wasn't updated for some reason. Can you rebuild and regenerate again to fix this?